addons/mail/static/src/models/thread.js

```diff
@@ -925,13 +926,7 @@ registerModel({
          * @param {Partner} partner
          */
         registerOtherMemberTypingMember(partner) {
-            const timer = new Timer(
-                this.messaging,
-                () => this._onOtherMemberLongTypingTimeout(partner),
-                60 * 1000
-            );
-            this.otherMembersLongTypingTimers.set(partner, timer);
-            timer.start();
+            this.update({ otherMembersLongTypingTimers: insert({ partner: replace(partner) }) });
             const newOrderedTypingMemberLocalIds = this.orderedTypingMemberLocalIds
```

Comment:
note: `orderedTypingMemberLocalIds`, `isTyping` and other such fields can probably be removed in favor of the existence of the timer(s)